indico/core/plugins/__init__.py

```diff
@@ -116,6 +116,9 @@ def _import_models(self):
         # which could contain more than one plugin this is not easily possible.
         for name, model in added_models:
             schema = model.__table__.schema
+            # Allow models with non-plugin schema if they specify `polymorphic_identity` without a dedicated table
+            if not hasattr(model, 'tablename') and 'polymorphic_identity' in getattr(model, '__mapper_args__', ()):
```

Comment:
is it `tablename` or `__tablename__`? can the model have a `Non` tablename? If yes maybe better to check getattr

Reply:
> can the model have a Non tablename?

True, this is not possible. I'm trying to account here for the possibility to use `polymorphic_identity` for [joined table inheritance](https://docs.sqlalchemy.org/en/13/orm/inheritance.html#joined-table-inheritance). In which case, you probably want the plugin to define the new table in its own plugin schema.

Reply:
`'__tablename__' in model.__dict__` should do the job